students/Roy_Tate/lesson04/trigrams.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+#  Author/Student: Roy Tate (githubtater)
+
+import string
+import random
+
+# variable to hold the location of the file to read. Place in current directory. Modify filename as needed.
+book_to_read = './sherlock.txt'
+
+def read_book():
+    '''Reads the text of a file and returns it in a list'''
+    stripped_text = []
+    with open(book_to_read) as f:
+        for line in f:
+            line.rstrip()  # remove newline chars
+            for char in line:
+                line = line.replace('--', ' ')  # replace chars
+                line = line.replace('-', ' ')
+            words = line.split()
+            for word in words:
+                stripped_text.append(word)
+    return stripped_text
+
+
+def create_words_dict(text):
+    '''Look at each set of three adjacent words in a document. Use the first two words of the
+    set as a key, and remember the fact that the third word followed that key.'''
+    stripped_text = ' '.join(text)
+    words_dict = {}
+    for count in range(0, len(stripped_text.split()) - 2):
+        first_word = stripped_text.split()[count]
+        second_word = stripped_text.split()[count+1]
+        follow_word = stripped_text.split()[count+2]
+        if (first_word, second_word) in words_dict:
```

Comment:
this would be good to have in a variable
`word_pair = (first_word, second_word)`